### ISSUE TITLE
Replay UI has UI/UX issues on mobile

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#495]
In testing a deployed instance of ipwb at ipwb.ws-dl.cs.odu.edu and found the replay interface to be difficult to use. This being the first opportunity we have been able to test the interface from a remote machine, much less from a mobile device, is understandable.

A few things that should be fixed:
- [ ] Links to URI-Ms are too close together and difficult to precisely touch
- [ ] Text elements in admin section are inconsistently sized (related: #213)
- [ ] Opening UI to show available mementos causes page to expand (as shown) and other interface items to be downscaled 
- [ ] ...quite a few other issues to be enumerated...

![screenshot_brave_20180811-121614 1](https://user-images.githubusercontent.com/2514780/43993772-bbdd786a-9d60-11e8-8d70-c01be65279af.png)

**Comment by ibnesayeed** [comment]
Moving admin stuff in in separate page would get rid of some noise from the landing page and to make the layout easier to manage we can utilize the CSS Grid.

**Comment by machawk1** [comment]
Even after we move admin stuff to a separate page (#213), both the main replay page and the admin page will need style adjustments to account for mobile. As we discussed in-person, utilization of CSS Grid would be one such solution.
